website/blog/2024-03-18-deep-learning-part-2/index.md

```diff
@@ -0,0 +1,230 @@
+---
+title: Fine Tuning LLMs with MLflow: Deep Learning with MLflow (Part 2)
+description: Highlighting the recent improvements in MLflow for Deep Learning workflows
+slug: deep-learning-part-1
+authors: [abe-omorogbe, avinash-sooriyarachchi, puneet-jain]
+tags: [Deep Learning]
+thumbnail: img/blog/dl-chart-grouping.gif
+---
+
+![](deep-learning-pt2-min.jpg)
+
+In the realm of deep learning, finetuning of pre-trained Large Language Models (LLMs) on private datasets stands as a critical application. This practice is not only common but also essential for developing specialized models, particularly for tasks like custom Named Entity Recognition (NER). In such scenarios, tools like MLflow are invaluable. They ensure that every aspect of the training process - metrics, parameters, and artifacts - are reproducibly tracked, logged, analyzed, compared, and shared. In this tutorial we are going to be using [MLflow 2.11](https://mlflow.org/releases/2.11.0) and [the new MLflow Deep Learning features](https://mlflow.org/blog/deep-learning-part-1) to track all the important aspects of this project.
+
+
+# Use Case: Fine Tuning Llama2-7B for NER on Restaurant data
+
```

Comment:
PEFT should be explained here as there are references below to benefits and functionality within the huggingface library that are incorrectly assumed that a reader would have. 2-3 sentences that explain what it is and how it is a cost-effective and generally more accurate means of performing model customization would be an effective primer to the rest of the blog.